tests/testthat/test-derive_var_dthcaus.R

```diff
@@ -33,17 +33,14 @@ test_that("derive_var_dthcaus Test 2: DTHCAUS is added from AE and DS", {
     "TEST01", "PAT01", 1, "INFORMED CONSENT OBTAINED", "INFORMED CONSENT OBTAINED", "2021-04-01",
     "TEST01", "PAT01", 2, "RANDOMIZATION", "RANDOMIZATION", "2021-04-11",
     "TEST01", "PAT01", 3, "ADVERSE EVENT", "ADVERSE EVENT", "2021-12-01",
-    "TEST01", "PAT01", 4, "DEATH", "DEATH DUE TO PROGRESSION OF DISEASE", "2022-02-01",
+    "TEST01", "PAT01", 4, "DEATH", "DEATH DUE TO progression of disease", "2022-02-01",
```

Comment:
does this lower case matter?

Reply:
Yes, it demonstrates that expressions for `dthcaus` work (see line 58: `dthcaus = str_to_upper(DSTERM)`).